### ISSUE TITLE
Fix the CSS for "Select Countdown" sidebar

### ISSUE THREAD

**Issue by willgarrett64** [MK-Codecademy/CountdownApp#31]
The select countdown sidebar has been built, but the styling has only been half complete. 

Note that the sidebar in the selectCountdownBar.html seems empty, but this is because the file createCountdownItem.js is handling creating and appending the countdown cards. This will eventually be linked up with data coming from the backend. 

The main styles that need fixing are:

- [ ] main header font size/weighting/line-height/letter-spacing/etc
- [ ] countdown card font size/etc
- [ ] better positioning of the time/date sections
- [ ] Add New card needs the dashed border fixing (currently a placeholder red border)
- [ ] background colours
- [ ] all hover effects

Basically, it all needs fixing.

![image](https://user-images.githubusercontent.com/74154911/139089784-af10d371-6092-4c32-abe8-63a4751e8b96.png)
